site/examples/iframe.tsx

```diff
@@ -104,15 +104,17 @@ const MarkButton = ({ format, icon }) => {
   )
 }
 
-const IFrame = ({ children, ...props }) => {
-  const [contentRef, setContentRef] = useState(null)
-  const mountNode =
-    contentRef &&
-    contentRef.contentWindow &&
-    contentRef.contentWindow.document.body
+const IFrame = ({ children, onLoad = null, ...props }) => {
```

Comment:
Any specific reason to assign null to `onLoad`, instead of leaving it undefined?

```suggestion
const IFrame = ({ children, onLoad, ...props }) => {
```

Reply:
Thanks for pointing that out. I was experimenting with solutions and I had to left out it there. And actually that whole `onLoad` is not needed there... I will remove it